src/rqt_graph/rosgraph2_impl.py

```diff
@@ -44,18 +42,17 @@
 import random
 import logging
 import traceback
-try:
-    from xmlrpc.client import ServerProxy
-except ImportError:
-    from xmlrpclib import ServerProxy
 import socket
 
-import rosgraph.masterapi
+from collections import defaultdict
 
-logger = logging.getLogger('rosgraph.graph')
+from python_qt_binding.QtCore import qDebug, qWarning
+from rclpy import logging
+logger = logging.get_logger('rqt_graph.rosgraph2_impl.graph')
```

Comment:
It would be better to name this `_logger` so that it doesn't get included with `import *`

Reply:
Two lines after imports